pkg/kotsadm/types/constants.go

```diff
@@ -52,3 +56,8 @@ func MergeLabels(labels ...map[string]string) map[string]string {
 
 	return mergedLabels
 }
+
+func LabelsToStrings(labelMap map[string]string) string {
```

Comment:
this functions seems redundant to me. since it really replaces a single line? can't the caller just do `labels.SelectorFromSet(labelMap).String()`?

Reply:
good idea, removed